src/SUMMARY.md

```diff
@@ -1,64 +1,105 @@
+- [How to](how_to.md)
+    * [Create a wallet]()
+ 
+<!-- Tutorials -->
 
-<!-- Smart contracts -->
+- [Tutorials](miden-tutorials/index.md)
+  - [Miden Node Setup](miden-tutorials/miden_node_setup.md)
+  - [Rust-Client](miden-tutorials/rust-client/about.md)
+    - [Creating Accounts and Faucets](miden-tutorials/rust-client/create_deploy_tutorial.md)
+    - [Mint, Consume, and Create Notes](miden-tutorials/rust-client/mint_consume_create_tutorial.md)
+    - [Deploying a Counter Contract](miden-tutorials/rust-client/counter_contract_tutorial.md)
+    - [Interacting with Public Smart Contracts](miden-tutorials/rust-client/public_account_interaction_tutorial.md)
+    - [Foreign Procedure Invocation](miden-tutorials/rust-client/foreign_procedure_invocation_tutorial.md)
 
-- [Smart contracts]()
-    * [Miden SDK]()
-    * [Examples]()
+  - [Web-Client](miden-tutorials/web-client/about.md)
+    - [Creating Accounts and Faucets](miden-tutorials/web-client/create_deploy_tutorial.md)
+    - [Mint, Consume, and Create Notes](miden-tutorials/web-client/mint_consume_create_tutorial.md)
 
-<!-- Node -->
+ <!-- Examples -->
 
-- [Node](miden-node/index.md)
- 
-<!-- Client -->
- 
-- [Client](miden-client/index.md)
-    * [Installation](miden-client/install-and-run.md)
-    * [Features](miden-client/features.md)
-    * [Design](miden-client/design.md)
-    * [Library](miden-client/library.md)
-    * [CLI]()
-        + [Reference](miden-client/cli-reference.md)
-        + [Config](miden-client/cli-config.md)
-    * [Examples](miden-client/examples.md)
-    * [API](miden-client/api-docs.md)
+- [Examples](examples.md)
+    * [Voting contract]()
 
  <!-- Protocol -->
 
-- [Protocol](miden-base/architecture/overview.md)
-    * [Smart contract](miden-base/architecture/account.md)
-    * [Note](miden-base/architecture/note.md)
-    * [Asset](miden-base/architecture/asset.md)
-    * [Transaction](miden-base/architecture/transaction.md)
-    * [State](miden-base/architecture/state.md)
-    * [Execution](miden-base/architecture/execution.md)
-
-<!-- Tutorials -->
-
-- [Tutorials](./miden-tutorials/index.md)
-  - [Miden Node Setup](./miden-tutorials/miden_node_setup.md)
-  - [Rust-Client]()
-    - [Creating Accounts and Faucets](./miden-tutorials/rust-client/create_deploy_tutorial.md)
-    - [Mint, Consume, and Create Notes](./miden-tutorials/rust-client/mint_consume_create_tutorial.md)
-    - [Deploying a Counter Contract](./miden-tutorials/rust-client/counter_contract_tutorial.md)
-    - [Interacting with Public Smart Contracts](./miden-tutorials/rust-client/public_account_interaction_tutorial.md)
-    - [Foreign Procedure Invocation](./miden-tutorials/rust-client/foreign_procedure_invocation_tutorial.md)
-
-  - [Web-Client]()
-    - [Creating Accounts and Faucets](./miden-tutorials/web-client/create_deploy_tutorial.md)
-    - [Mint, Consume, and Create Notes](./miden-tutorials/web-client/mint_consume_create_tutorial.md)
+- [Protocol](protocol.md)
```

Comment:
I'm not sure the sections below are really about the protocol. Miden VM, client and node should probably be their own top-level sections. "Protocol" in my mind describes the blockchain protocol (what we currently have in the "Concepts" section).